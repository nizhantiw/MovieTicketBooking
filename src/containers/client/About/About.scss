.about{
    background-color: lightcyan;
    height: 100vh;
    display: flex;
    justify-content: center;
    top: 40px;
}
h3{

    width: 80%;
}