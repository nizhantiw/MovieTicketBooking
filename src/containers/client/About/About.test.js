import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer'
import About from './About'
configure({adapter: new Adapter()});

it("renders correctly", () => {
  const wrapper = shallow(
    <About />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders a snapshot', () => {
  const tree = renderer.create(<About/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('should show the text', () => {
    const toggleInstance = shallow(<About />);
    const element = toggleInstance.find('div h3');
    expect(element.text()).toMatchSnapshot();
   });

   it('renders results after search', () => {
    const wrapper = shallow(<About/>);
    expect(wrapper.find('img').length).toEqual(1);
  });
  it('starts with a count of 0', () => {
      const wrapper = shallow(<About />);
      const text = wrapper.find('h3').text();
      expect(text).toEqual('Movie Tickets is an online movie ticketing website founded by AMC Theatres and Hollywood.com in 2000; CBS Corporation, Famous Players, and National Amusements all came on board prior to launch;[1] and it is now a subsidiary of Fandango Media. MovieTickets.com provides movie times for all theaters, and online ticket purchasing for all Clearview Cinemas and National Amusements theaters, among other smaller chains; such as Mann Theatres in Los Angeles. In 2010, MovieTickets.com sold over 16 million tickets for over 200 exhibitors, with 14,000 screens.');
    });

