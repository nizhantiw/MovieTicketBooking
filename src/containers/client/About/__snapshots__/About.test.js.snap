// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="about"
>
  <img
    alt="images"
    src="/carousal/aa.jpg"
    style={
      Object {
        "height": "570px",
      }
    }
  />
  <br />
  <h3>
    <br />
    Movie Tickets is an online movie ticketing website founded by AMC Theatres and Hollywood.com in 2000; CBS Corporation, Famous Players, and National Amusements all came on board prior to launch;[1] and it is now a subsidiary of Fandango Media. MovieTickets.com provides movie times for all theaters, and online ticket purchasing for all Clearview Cinemas and National Amusements theaters, among other smaller chains; such as Mann Theatres in Los Angeles. In 2010, MovieTickets.com sold over 16 million tickets for over 200 exhibitors, with 14,000 screens.
  </h3>
</div>
`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;

exports[`should show the text 1`] = `"Movie Tickets is an online movie ticketing website founded by AMC Theatres and Hollywood.com in 2000; CBS Corporation, Famous Players, and National Amusements all came on board prior to launch;[1] and it is now a subsidiary of Fandango Media. MovieTickets.com provides movie times for all theaters, and online ticket purchasing for all Clearview Cinemas and National Amusements theaters, among other smaller chains; such as Mann Theatres in Los Angeles. In 2010, MovieTickets.com sold over 16 million tickets for over 200 exhibitors, with 14,000 screens."`;
